src/components/Allocation/AddEditAllocationModal.tsx

```diff
@@ -229,6 +254,21 @@ export const AddEditAllocationModal = ({
               : t`Distribution Percentage`
           }
           required
+          extra={
+            showFee &&
+            amountAfterFee &&
+            ethPaymentTerminalFee && (
+              <Trans>
+                {amountAfterFee} after fee{' '}
+                <TooltipIcon
+                  tip={t`
+                      There is a ${formatFee(
```

Comment:
Lemme know about this copy @tomquirk

Reply:
@wraeth-eth lets steal all the copy from https://github.com/jbx-protocol/juice-interface/blob/c27300806df1409d7a140ca5e108bf16624626f3/src/components/v2v3/shared/FundingCycleConfigurationDrawers/DistributionSplitModal/AmountFormItem.tsx#L47-L67